docs/en/concepts-and-designs/oal.md

```diff
@@ -120,12 +120,6 @@ endpoint_percentile = from(Endpoint.latency).percentile(10)
 // Caculate the percent of response status is true, for each service.
 endpoint_success = from(Endpoint.*).filter(status == true).percent()
 
-// Caculate the percent of response code in [200, 299], for each service.
-endpoint_200 = from(Endpoint.*).filter(responseCode like "2%").percent()
-
-// Caculate the percent of response code in [500, 599], for each service.
-endpoint_500 = from(Endpoint.*).filter(responseCode like "5%").percent()
-
```

Comment:
I think we still should have some `like` examples, such as, `name like "namespace1.%"`? Uses may need to filter the services for the specific metrics. WDYT?

Reply:
![image](https://user-images.githubusercontent.com/26432832/94023729-2351eb80-fde9-11ea-9604-e2e9f8644c20.png)
There is already one.

Reply:
Oops, sorry.